src/pages/UserList/UserList.js

```diff
@@ -28,12 +58,15 @@ const usersQuery = {
             selfRegistered,
             nameSortDirection,
             organisationUnits,
+            emailVerificationStatus,
+            displayEmailVerifiedStatus,
         }) => ({
             fields: [
                 'id',
                 'displayName',
                 'access',
                 'email',
+                'emailVerified',
```

Comment:
this is now added to the request regardless, but will not be returned if it does not exist / will not be displayed if the conditions for this feature are not met.
I didn't think it was worth it to change the query definition based on the feature logic